recipes/RECOLA/preprocess.py

```diff
@@ -0,0 +1,224 @@
+import glob
+import json
+import os
+import torchaudio
+from pathlib import Path
+
+
+def preprocess_RECOLA(
+    data_path_audio,
+    data_path_arousal,
+    data_path_valence,
+    experiment_folder,
+    data_processed_folder,
+):
+    """
+    Transforms audio files and makes the json files for all the data and different partitions
+
+    Arguments
+    ---------
+    data_path_audio : str
+        The path to the folder containing RECOLA audio .wav files
+    data_path_arousal : str
+        The path to arousal annotations of RECOLA (2016 format .arff files)
+    data_path_valence : str
+        The path to valence annotations of RECOLA (2016 format .arff files)
+    experiment_folder : str
+        The experiments folder, used to store json files for partitions and other information
+    data_processed_folder : str
+        The folder containing the processed data
+
+    Example
+    -------
+
+    >>> preprocess_RECOLA("./RECOLA_2016/recordings_audio",
+                          "./RECOLA_2016/ratings_gold_standard/arousal",
+                          "./RECOLA_2016/ratings_gold_standard/valence",
+                          "./Experiments",
+                          "./Experiments/Audio")
+    """
+
+    # Avoiding reprocessing
+    if os.path.exists(os.path.join(experiment_folder, "data.json")):
+        print("RECOLA is already processed")
+        return
+
+    originalAudioPath = data_path_audio
+    jsonFolderPath = experiment_folder
+    arousalARFFolderPath = data_path_arousal
+    valenceARFFolderPath = data_path_valence
+    ProcessedAudioPath = data_processed_folder
```

Comment:
lower case: processedAudioPath
for variables better as: processed_audio_path ?
CamelCase is with classes usually, no?